### PR TITLE
Ba odd

### PR DESCRIPTION
**Pull request by bernardanderson**

### CODE REVIEW

**Comment by sylviama** [comment]
👍

File: Mathmagician.Tests/OddTests.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        [TestMethod]
+        public void Odd_ReturnedListLength()
+        {
+            //arrange (nothing in this case)
+
+            //act (nothing in this case)
+
+            //assert 
+            Assert.AreEqual(10000, new Odd().CountOddNumbers(10000).Count);
+        }
```
Comment (katerebekah) [inline Mathmagician.Tests/OddTests.cs:68]: Add a CollectionAssert.AreEqual test that will compare an expected list to the one your class returns.

File: Mathmagician/UserInterface.cs [hunk 2]
```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician
+{
+    public class UserInterface
```
Comment (katerebekah) [inline Mathmagician/UserInterface.cs:9]: You can unit test the methods in this class, too, but it might not be required.

**Comment by bernardanderson** [comment]
@katerebekah 
Thanks, I added the test you requested to the ba-odd branch.

**Comment by katerebekah** [comment]
:+1:

**Pull request merged**